man/systemd.network.xml

```diff
@@ -3192,6 +3192,14 @@
           to the class. Defaults to unset.</para>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>RateToQuantum=</varname></term>
+        <listitem>
+          <para>Takes an unsigned integer. The DRR quantums are calculated by <varname>Rate=</varname> /
```

Comment:
is that `/` supposed to mean division? if so, maybe spell this out as "divided by".

Maybe:

```
The DRR quantums are calculated by dividing the value configured in Rate= by RateToQuantum=.
```

or so